packages/perseus-editor/src/editor.tsx

```diff
@@ -183,11 +184,11 @@ class Editor extends React.Component<Props, State> {
         // this.props.onChange during that, since it calls our parent's
         // setState
         this._sizeImages(this.props);
-        // eslint-disable-next-line react/no-string-refs
-        // @ts-expect-error - TS2769 - No overload matches this call.
-        $(ReactDOM.findDOMNode(this.refs.textarea))
+        // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+        $(this.textarea.current!)
```

Comment:
jQuery doesn't like potential `undefined`, but this is exactly how the previous code was, and given this is in `componentDidMount`, we _should_ have a valid ref at this point.

Reply:
TIL about the TS non-null assertion operator. Is this pretty common in TS or would it be worth commenting the code? I guess the ESLint exception is a clue, but maybe why we're adding the exception?

Reply:
I don't think it's very common. I'm used to it as there is a similar operator in Swift that we sometimes used. 

I'll add a comment.